### DISCUSSION TITLE
free-water kurtosis

### DISCUSSION THREAD

**Discussion by amorelli00** [dipy/dipy#2869] [Reconstruction] [answered]
Hello Community,
I have worked for several year with DTI and free-water DTI.
Some weeks ago, I read the paper:
Diffusion Kurtosis Imaging With Free Water Elimination: A Bayesian Estimation Approach
https://pubmed.ncbi.nlm.nih.gov/29393531/

And I would like to know if someone know the algorithm and where I can find it.

Thanks,
Andrea

**Comment by skoudoro** [comment]
@shilpiprd, @RafaelNH , @arokem  are the team member who are used to work with DKI.

Can you answer to @amorelli00 ?

Thanks!

**Comment by shilpiprd** [comment]
I'm sorry actually, i haven't had the opportunity to read this particular paper yet and hence I can't provide any details about this specific algorithm.

**Comment by RafaelNH** [comment, marked as answer]
Hi Andrea. We don't have an implementation of free-water DKI in Dipy. Free-water DKI is an ill-posed problem in typical diffusion MRI data acquisition and can thus produce unreliable estimates. I haven't read the paper you mentioned in its full extent, but from the paper's abstract, you can note that the authors acknowledge that free-water DKI is ill-posed. Therefore, having it in Dipy would be challenging since it might not pass its code implementation testing phase.

**Comment by amorelli00** [comment]
Yes, I saw that it is a is an ill-posed problem. Thanks for you answer!
Andrea

**Comment by amorelli00** [comment]
Hello, again,
sorry but I have another question, because I am very interested to FW-DTI and FW_DKI.
I had an idea, but probably it might be a stupid idea, therefore, I need your opinion.
I have a multi-shell diffusion acquisition. After the pre-processing I run the Dipy FW-DTI:
fwdtimodel = fwdti.FreeWaterTensorModel(gtab)
fwdtifit = fwdtimodel.fit(data, mask=Mymask)

in this case I can have the FW index (f) and the corrected tensor (with eigenvectors and eigenvalues).
After this, I can run the function:
fwdti.fwdti_prediction

to have the simulated signal based on the free water DTI model.
And, I could use this new signal to run the DKI model and have the DKI-metrics corrected for FW.

In you opinion, is it a reasonable method?

Andrea

**Comment by RafaelNH** [comment]
That is not a stupid idea. However, the issue of your proposed method is that free water fraction can be misestimated by tissue non-Gaussian effects not considered by fwDTI (specially if you are using fwdti in data acquired with high b-values). Therefore, when you fit DKI in fwdti predicted signals you are propagating the free water biases to your kurtosis estimates. In addition, noise in our free water estimates will also be propagated to the DKI estimates which may worse the precision of DKI estimates.

**Reply by RafaelNH** [reply]
@amorelli00 - Just out of curiosity - what b-values here used for your data?

**Reply by amorelli00** [reply]
Hi, the b-values are 0, 1000, and 2000
